compiler/rustc_hir_analysis/src/hir_ty_lowering/errors.rs

```diff
@@ -719,9 +719,9 @@ impl<'tcx> dyn HirTyLowerer<'tcx> + '_ {
         associated_types: FxIndexMap<Span, FxIndexSet<DefId>>,
```

Comment:
existing, but `complain_about_missing_assoc_tys` is a weird name, why not simply `check_for_required_assoc_tys`